### PR TITLE
Add ConsoleMp typings

### PR DESCRIPTION
**Pull request by nnsdev**
RAGE 1.0 will feature a clientside console reachable under `mp.console`

This PR adds typings for it, also removes some whitespaces.

### CODE REVIEW

**Comment by pixlcrashr** [comment]
@JulienGelmar You maybe wait with your PR until a seperate branch "v1.0" is added to this repository or until v1.0 is released by RageMP, as the changes this PR provides are not compatible with the current version and thus the current available projects.

See #45 for reference.

**Comment by nnsdev** [comment]
Agreed, would target my PR toward this newly created branch as well.

File: index.d.ts [hunk 1]
```diff
@@ -175,7 +176,7 @@ interface EntityMp {
 		boneIndex: number, isRel: number, p9: boolean, highForce: boolean, p11: boolean, p12: boolean): void;
 	applyForceToCenterOfMass(forceType: number, x: number, y: number, z: number, p4: boolean, isRel: boolean,
 		highForce: boolean, p7: boolean): void;
-	attachTo(entity: Handle, boneIndex: number, xPos: number, yPos: number, zPos: number, xRot: number, yRot: number,
```
Comment (pixlcrashr) [inline index.d.ts:178]: What about seperate commits? Though I know I am a bit nit-picking by saying so but it is clearer for everybody what has changed (as your PR only says that you'll add the ConsoleMp typing). Nvm.

**Comment by pixlcrashr** [comment]
@JulienGelmar I just created branch `v1.0`. You now can resubmit the changes within the correct branch.

For now, I'll close this PR. If there is anything left to discuss, just reopen this PR.

**Pull request closed without merging**